requirements.in

```diff
@@ -7,6 +7,6 @@ gunicorn==20.1.0
 # PaaS
 awscli-cwlogs==1.4.6
 
-git+https://github.com/alphagov/notifications-utils.git@74.8.0
+git+https://github.com/alphagov/notifications-utils.git@74.9.0
```

Comment:
Will this need bumping up to 74.9.1 before it gets deployed so it brings in https://github.com/alphagov/notifications-utils/pull/1097/files

I don't know if the celery task args and kwargs are sensitive or not for the tasks that the antivirus worker runs, but not worth taking the risk of course...